src/index.html

```diff
@@ -7,8 +7,63 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__links">
+
+          <div class="rectangle"></div>
+
+          <li class="nav__li">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#Laptops & Computers"
+              class="nav__link hover"
```

Comment:
The class 'hover' is not relevant according to the task. The hover effect should be applied to all links, not just one. Consider removing this class or ensuring that the hover effect is applied consistently across all links.